free-1.md

```diff
@@ -57,12 +82,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think the time spent would be evenly split between figuring out how to make this into a comprehensive language, and doing user studies to figure out what's important to people that's also feasible to calculate outside of the game. The actual calculations the game uses are fairly well-known, so its pretty clear how to make the backend work. However, there would be a good amount of time spent inputting all of the data for the existing heroes, but it would be a one-time setup cost, not a thing to continue devloping over time. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think its fairly ambitious. There's a lot of raw data I'd need to collect, as well as a bunch of research that would go into the backend, and the frontend needs to be very, very user-friendly for the average person to be willing to learn it. However, all of this seems very doable, there's nothing in it thats like "wow, I actually don't know how I would do that". Its just a lot of work to be done. 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a very useful tool for a very specific domain. Also, while its a lot of work to input the basic data for the backend, it would also be very easy to just set it up for a subset of the heroes/items. The language could be developed modularly, starting with some data and some output (like just hero damage at first), and then the implementation could grow naturally over time.
+
+However, the domain is possibly too specific. The majority of people don't know DOTA 2, and of the people who do, many just play it casually, so wouldn't care too much about something thats intended to make them better at the game. There's also a large amount of data that needs to get collected for the thing to even work. 
+
```

Comment:
I wouldn't say that the domain is too specific.  Dota2 has at least [a million](http://steamcharts.com/app/570) active players, and I'm sure a large portion of them would be interested in something like this.